common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidDeviceMetadata.java

```diff
@@ -56,10 +56,11 @@ public String getCpu() {
         return "UNKNOWN";
     }
 
+    // Returns a SDK version (i.e. 24) instead of a user-friendly android version (i.e. 7.0)
```

Comment:
Do we want to support the other concept as well?

Reply:
I prefer not to do that unless we really have to. Will check with satheesh and co.